src/Artsy/Router/RouterLink.tsx

```diff
@@ -0,0 +1,23 @@
+import { Link, LinkProps } from "found"
+import React from "react"
+
+/**
+ * Wrapper component around found's <Link> component that can be easily styled
+ * via styled-components; e.g.,
+ *
+ * const StyledLink = styled(RouterLink)`
+ *   ...
+ * `
+ */
+export const RouterLink: React.FC<LinkProps> = ({
```

Comment:
So should we use this over `import { Link } from "@artsy/palette"` from now on?

Reply:
If we need to navigate within the router framework to other `/collections` / `collection/id`, `/collect` pages, yup. Can also use `import { Link } from 'found'` if not needing to style fancy.